### ISSUE TITLE
Not able to access Mighty Text

### ISSUE THREAD

**Issue by gnarfle** [EFForg/privacybadger#149]
[MightyText](mightytext.net) uses a web based app to read phone text messages on your computer. With privacy badger enabled I can't use the site, probably because it authenticates with google and when you visit the web app you get redirected a few times. I've tried to disable privacy badger for that site but have not been able to because the redirects happen quickly and prevent you from being able to disable privacy badger for all the URL's it bounces through.

In short, when trying to use their web app with privacy badger enabled it ends up with an endless redirect loop and I haven't found any way to prevent this other than completely disabling privacy badger.
